charts/sourcegraph/values.yaml

```diff
@@ -322,6 +322,52 @@ embeddings:
   extraVolumeMounts: {}
   extraVolumes: {}
 
+qdrant:
+  # -- Enable `qdrant`
+  enabled: false
+  # -- Name of the `qdrant` service
+  name: qdrant
+  image:
+    # -- Docker image name for the `embeddings` image
+    name: "qdrant"
+    # -- Docker image tag for the `embeddings` image
+    defaultTag: "239247_2023-08-18_5.1-433e1b1c997f@sha256:eafcd7af2aca699fa9c9ce8e6aa674cc0470441f794baf031296d5d1cdadd0bc"
+  # -- Resource requests & limits for the `qdrant` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  resources:
+    limits:
+      cpu: "4"
+      memory: 16G
+    requests:
+      cpu: "4"
+      memory: 16G
+  # -- Environment variables for the `qdrant` container
+  env: {}
+  # -- Security context for the `qdrant` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
+  containerSecurityContext:
+    allowPrivilegeEscalation: false
+    runAsUser: 100
+    runAsGroup: 101
+    fsGroup: 101
+  # -- Security context for the `qdrant` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
+  podSecurityContext:
+    runAsUser: 100
+    runAsGroup: 101
+    fsGroup: 101
+    fsGroupChangePolicy: "Always"
+  serviceAccount:
+    # -- Enable creation of ServiceAccount for `embeddings`
+    create: false
+    # -- Name of the ServiceAccount to be created or an existing ServiceAccount
+    name: ""
+    annotations: {}
+  extraVolumeMounts: {}
+  extraVolumes: {}
+  # -- PVC Storage Request for `qdrant` data volume
+  storageSize: 100Gi
```

Comment:
How difficult is it to increase the size of a PVC?

Reply:
It looks like we [allow volume expansion](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-helm@7b2814d123c1739a29ebc0ede47097fe35608634/-/blob/charts/sourcegraph/templates/storageclass.yaml?L21-21) by default, but I'm honestly not sure what the ramificaitons are. 100Gi is large enough for a large chunk of customers, but certainly not all.

Reply:
cloud prefers to start small and scale up as needed (scaling up is easy and we have monitoring for it)